x-pack/plugins/observability_solution/inventory/public/plugin.ts

```diff
@@ -91,7 +91,7 @@ export class InventoryPlugin
         defaultMessage: 'Inventory',
       }),
       euiIconType: 'logoObservability',
-      appRoute: '/app/observability/inventory',
```

Comment:
Q: Should we handle a redirect here? The plugin was released just recently I don't expect  many user to bookmark this already

Reply:
No, it's a tech preview feature.